File: Beginner/Fibonacci_Series.py
def fib_seq(n):
    fib_ser="1"
    fib_otro="0"
    n=int(n)
    if n>2:
        series = fib_otro +","+fib_ser
        for i in range(1, n-1):
            sum=int(fib_ser)+int(fib_otro)
            fib_otro=fib_ser
            fib_ser=sum
            series = series + "," + str(fib_ser)
    elif n==1:
        series="0"
    elif n==2:
        series="0,1"
    return series

n=""
while not n.isdigit():
    n=input("Enter your Number: ")
    if n.isdigit():
        if int(n)<1:
            n=""
print(fib_seq(n))
input("Enter")

File: BEGINNER/fibonacci_series.cpp
#include<iostream>
using namespace std;
int main(){
  int n;
  cout<<"Enter the number of terms: "<<endl;
  cin>>n;
  int a=0,b=1;
  if(n==1){
    cout<<a<<endl;
  }
  else if(n==2){
    cout<<a<<" "<<b<<endl;
  }
  else{
    cout<<a<<" "<<b<<endl;
    for(int i=0;i<n-2;i++){
      c=a+b;
      a=b;
      b=c;
      cout<<" "<<c<<endl;
    }
  }
}
